src/main/java/seedu/duke/command/ListCommand.java

```diff
@@ -10,6 +10,7 @@
      * @param list the ItemContainer to retrieve list of items
      */
     public void execute(ItemContainer list) {
+        assert list != null : "List should not be null";
```

Comment:
Wait this should be an exception, not an assertion. JVM does not enable assertion by default, so this may check at all. Only use assertion to check the invariance in your code, such as after a series of operation, a variable should change from 10 to 0, then you assert this variable = 0.